MathExp/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace MathExp
+{
+    class Program
+    {
+        static void Main()
+        {
+            var result = Math.Round((Power(2, 3) + Power(3, 5) * Power(5, 7)), 7);
+            Console.WriteLine($"Значение выражения равно: {result}");
+            Console.ReadKey();
+        }
+        static double Power(int a, int b)
+        {
+            
+            var sideres = Math.Exp(-Math.Sqrt(a + (Math.Pow(b, 2))));
```

Comment:
В квадрат возводить лучше с помощью произведения\
```suggestion
            var sideres = Math.Exp(-Math.Sqrt(a + (b * b)));
```